try.html

```diff
@@ -714,6 +714,14 @@ <h3 id="miscellaneous"> Longer Miscellaneous </h3>
     <td><img src='/ansible/role/3078.svg' alt=''/></td>
     <td><code>https://img.shields.io/ansible/role/3078.svg</code></td>
   </tr>
+  <tr><th> Libraries.io for releases: </th>
+    <td><img src='https://img.shields.io/librariesio/release/hex/phoenix/1.0.3.svg' alt=''/></td>
```

Comment:
There should not be an `https://img.shields.io` at the start of the URL here, otherwise we cannot test the website.

I corrected it in f84b5cd.